js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useDefinitionTagFilter.tsx

```diff
@@ -94,23 +93,6 @@ export function useTagsForObjects<T>(
   );
 }
 
-export function doesFilterArrayMatchValueArray<T, V>(
```

Comment:
Moved this to a separate file so that the webworker doesnt end up pulling React.